services/dy-jupyter-extensions/osparc-lab/requirements/prod.txt

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
This file is empty because ``base.txt`` is empty! Therefore osparc-lab has no requirements at all

Reply:
That is correct. I have to talk to AMC to make sure what additions he needs and then I will add them there. So far, only the base images requirements are in.